.merlin

```diff
@@ -20,6 +20,7 @@ B _build/lib/bap_plugins
 B _build/lib/text_tags
 B _build/lib/regular
 B _build/lib/graphlib
+B _build/lib/microx
```

Comment:
this is a bad line... It will refer to a library visible via the findlib, not to the library compiled in `_build`. So consider the following: you're updating a library, and didn't reinstall it yet, because, plugin, depending on the library doesn't compile. In that case, Merlin will see the old (installed one) library, not the updated, where the compiler will see that library that is in the `_build`. Bad things may happen. 

I'm actually, currently updating merlin files, and trying to impose some order on them. So it is better not to touch them. Otherwise we will have lots of conflicts :)